### ISSUE TITLE
resource elasticsearch_index_template doesn't support import

### ISSUE THREAD

**Issue by bquartier** [phillbaker/terraform-provider-elasticsearch#39]
Hi,
First, thanks for your work on this provider :)

When importing index templates an error is thrown saying that:
```Error: resource elasticsearch_index_template doesn't support import```

Is it expected? Is there any plan to make index templates importable (or reason to not do it)?

**Comment by phillbaker** [comment]
Hi @bquartier, happy to see it's getting some use. 

It looks like we missed added the importer into quite a few resources, including the index template. See https://github.com/phillbaker/terraform-provider-elasticsearch/blob/ef8f5d91ce06b739d5eff637811a48f0f19ecc35/resource_elasticsearch_index_template.go#L14

Usually it's as straightforward as adding these lines: https://github.com/phillbaker/terraform-provider-elasticsearch/blob/ef8f5d91ce06b739d5eff637811a48f0f19ecc35/resource_elasticsearch_destination.go#L33-L35

However, we should also add a test as I've seen things not quite go as expected sometimes. Would you like to submit a PR?

**Comment by bquartier** [comment]
Hi @phillbaker ,

I have 0 experience in Go, but I can have a look and provide a PR if you provide me a few hints.

I need to find time and investigate how the tests work. It seems there is only one example of import test, in resource_elasticsearch_destination_test.go, do you confirm I can start from this example? The name of the function (TestAccElasticsearchDestination_importBasic) looks non standard; is it the way to go?

**Comment by bquartier** [comment]
A first hint regarding how to successfully run the tests would be great.
After setting the expected environment variables, I started the docker-compose ans launch the tests, however, I get errors saying that "no Elasticsearch node available".

See below:
```shell
✗ curl http://elastic:elastic@127.0.0.1:9210
{
  "name" : "elasticsearch-xpack",
  "cluster_name" : "xpack",
  "cluster_uuid" : "UT29-ayqTyilrBTnw0OYwg",
  "version" : {
    "number" : "7.4.2",
    "build_flavor" : "default",
    "build_type" : "docker",
    "build_hash" : "2f90bbf7b93631e52bafb59b3b049cb44ec25e96",
    "build_date" : "2019-10-28T20:40:44.881551Z",
    "build_snapshot" : false,
    "lucene_version" : "8.2.0",
    "minimum_wire_compatibility_version" : "6.8.0",
    "minimum_index_compatibility_version" : "6.0.0-beta1"
  },
  "tagline" : "You Know, for Search"
}

curl http://elastic:elastic@127.0.0.1:9220
{
  "name" : "elasticsearch-opendistro",
  "cluster_name" : "opendistro",
  "cluster_uuid" : "hWjuaZZCQNmAPEfMuY7VVQ",
  "version" : {
    "number" : "7.2.1",
    "build_flavor" : "oss",
    "build_type" : "tar",
    "build_hash" : "fe6cb20",
    "build_date" : "2019-07-24T17:58:29.979462Z",
    "build_snapshot" : false,
    "lucene_version" : "8.0.0",
    "minimum_wire_compatibility_version" : "6.8.0",
    "minimum_index_compatibility_version" : "6.0.0-beta1"
  },
  "tagline" : "You Know, for Search"
}
```
 So elasticsearches are up and running. However, I get a lot of `SKIP` when running the tests:
```shell
✗ go test -v
=== RUN   TestAccElasticsearchDataSourceDestination_basic
--- SKIP: TestAccElasticsearchDataSourceDestination_basic (5.00s)
    data_source_elasticsearch_destination_test.go:18: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchDataSourceHost_basic
--- SKIP: TestAccElasticsearchDataSourceHost_basic (0.00s)
    testing.go:462: Acceptance tests skipped unless env 'TF_ACC' set
=== RUN   TestProvider
--- PASS: TestProvider (0.00s)
=== RUN   TestProvider_impl
--- PASS: TestProvider_impl (0.00s)
=== RUN   TestAccElasticsearchDestination
--- SKIP: TestAccElasticsearchDestination (5.00s)
    resource_elasticsearch_destination_test.go:20: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchDestination_importBasic
--- SKIP: TestAccElasticsearchDestination_importBasic (5.00s)
    resource_elasticsearch_destination_test.go:57: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchIndexLifecyclePolicy
--- SKIP: TestAccElasticsearchIndexLifecyclePolicy (5.00s)
    resource_elasticsearch_index_lifecycle_policy_test.go:22: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchIndexTemplate
--- SKIP: TestAccElasticsearchIndexTemplate (5.00s)
    resource_elasticsearch_index_template_test.go:21: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchIngestPipeline
--- SKIP: TestAccElasticsearchIngestPipeline (5.00s)
    resource_elasticsearch_ingest_pipeline_test.go:21: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchKibanaObject
--- SKIP: TestAccElasticsearchKibanaObject (5.00s)
    resource_elasticsearch_kibana_object_test.go:22: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchMonitor
--- SKIP: TestAccElasticsearchMonitor (5.00s)
    resource_elasticsearch_monitor_test.go:20: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchSnapshotRepository
--- SKIP: TestAccElasticsearchSnapshotRepository (0.00s)
    testing.go:462: Acceptance tests skipped unless env 'TF_ACC' set
=== RUN   TestAccElasticsearchWatch
--- SKIP: TestAccElasticsearchWatch (5.00s)
    resource_elasticsearch_watch_test.go:21: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchXpackRoleMapping
--- SKIP: TestAccElasticsearchXpackRoleMapping (5.00s)
    resource_elasticsearch_xpack_role_mapping_test.go:22: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchXpackRole
--- SKIP: TestAccElasticsearchXpackRole (5.00s)
    resource_elasticsearch_xpack_role_test.go:22: err: health check timeout: no Elasticsearch node available
=== RUN   TestAccElasticsearchXpackUser
--- SKIP: TestAccElasticsearchXpackUser (5.00s)
    resource_elasticsearch_xpack_user_test.go:22: err: health check timeout: no Elasticsearch node available
PASS
ok  	github.com/phillbaker/terraform-provider-elasticsearch	60.045s
```

Any hint?

**Comment by phillbaker** [comment]
Sorry for the slow response, that's odd! It does look like you need to set `TF_ACC=1` similarly to travis.yml

**Comment by phillbaker** [comment]
Fixed in 92b143a.
